test/index.ts

```diff
@@ -1,25 +1,59 @@
-import { fastify } from 'fastify';
-import { runServer } from '../src/server';
-import dotenv from 'dotenv';
 import axios from 'axios';
+import * as chai from 'chai';
+import prismaInstance from '../src/db';
+import { closeServer, runServer } from '../src/server';
+import { UserReq } from '../src/user/user.model';
 
-dotenv.config({ path: '../.test.env' });
-const app = fastify();
-const port: number = process.env.PORT || 8081;
-const server = axios.create({
+const port: number = process.env.PORT ? parseInt(process.env.PORT, 10) : 8081;
+const endpoint = axios.create({
   baseURL: `http://localhost:${port}`,
 });
+const input: UserReq = {
+  name: 'Gabrielle',
+  email: 'gabi@gmail.com',
+  password: 'coxinha123',
+  birthDate: '02-12-2000',
+};
 
-before(() => {
-  app.get('/hello', () => {
-    return 'hello, world!';
-  });
-  runServer(app, port);
+before(async () => {
+  try {
+    await runServer(port);
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
 });
 
 describe('Hello, world endpoint', () => {
-  it('return hello, world', async () => {
-    const response = await server.get('/hello');
-    console.log(response.data);
+  it(`return "hello, world!"`, async () => {
+    const response = await endpoint.get('/hello');
+    chai.expect(response.data).to.be.eq('hello, world!');
+  });
+});
+
+describe('Create user endpoint', () => {
+  it('Response and create sucess', async () => {
+    const response = await endpoint.post('/users', input);
+    chai.expect(response.status).to.be.deep.eq(201);
+    const userCreated = await prismaInstance.user.findUnique({
+      where: {
+        email: input.email,
+      },
+    });
```

Comment:
```suggestion
    const userCreated = (await prismaInstance.user.findUnique({
      where: {
        email: input.email,
      },
    }))!;
```

Só uma técnica pra vc não precisar repetir os `?` abaixo disso.